packages/plugin-core/src/commands/Sync.ts

```diff
@@ -73,38 +86,123 @@ export class SyncCommand extends BasicCommand<CommandOpts, CommandReturns> {
     const message = ["Finished sync."];
 
     // Report anything unusual the user probably should know about
-    const uncommitted = SyncCommand.filteredRepoNames(
-      committed,
-      SyncActionStatus.UNCOMMITTED_CHANGES
-    ).join(", ");
-    if (uncommitted.length > 0) {
-      message.push(
-        `Skipped pulling repos ${uncommitted} because they have uncommitted changes.`
-      );
-    }
-    const noUpstream = _.uniq([
-      ...SyncCommand.filteredRepoNames(pushed, SyncActionStatus.NO_UPSTREAM),
-      ...SyncCommand.filteredRepoNames(pulled, SyncActionStatus.NO_UPSTREAM),
-    ]).join(", ");
-    if (noUpstream.length > 0) {
-      message.push(
-        `Skipped pulling or pushing repos ${noUpstream} because they don't have upstream branches configured.`
-      );
-    }
+    let maxMessageSeverity: MessageSeverity = MessageSeverity.INFO;
+    const makeMessage = (
```

Comment:
nitpick, probably worhwile to extract this as a separate function that encapsulates all `makeMessage` calls